### ISSUE TITLE
pandasgui shutdown out of interactive cmd, pandasgui.show(df, settings={'block': True}) is needed.

### ISSUE THREAD

**Issue by forhonourlx** [adamerose/pandasgui#65]
Hi @adamerose ,

I am trying pandasgui in some python script, unlike in interactive cmd, it shutdown immediately after running.
I find `settings={'block': True}` is necessary, would you please figure it out in the demo examples?
Thank you and pandasgui is nice tool!

**Comment by adamerose** [comment]
It's working properly for me:

![image](https://user-images.githubusercontent.com/10884874/98461815-5247d500-217d-11eb-959e-a83c58409138.png)

I have some logic to make it so `settings={'block': True}` isn't needed in most cases, this is it:

```
def in_interactive_console():
    # https://stackoverflow.com/a/64523765/3620725
    return hasattr(sys, 'ps1')

if in_interactive_console():
    # Don't block if in an interactive console (so you can view GUI and still continue running commands)
    block = False
else:
    # If in a script, we need to block or the script will continue and finish without allowing GUI interaction
    block = True
```


How are you running the script? The only test case where it doesn't predict properly is when I run a script from PyCharm with this option checked, otherwise it always blocks properly

![image](https://user-images.githubusercontent.com/10884874/98461839-84f1cd80-217d-11eb-9589-f07afdaf61c0.png)

**Comment by forhonourlx** [comment]
Hi,

I am running the following code on win10 anaconda python3.6, pandasgui.show() shutdown immediately without a block. 
```
import pandas as pd
import pandasgui

class Render(object):
    def __init__(self, xlsx):
        self.store = pd.read_excel(xlsx, sheet_name=None)

if __name__ == '__main__':
    cr = Render('aaaa.xlsx')
    pandasgui.show(**cr.store)
```

**Comment by adamerose** [comment]
Run `pip install --upgrade pandasgui` to make sure you have the latest version.

> I am running the following code on win10 anaconda python3.6

But what method are you actually using to run your script, are you typing `python myscript.py` in Anaconda Prompt or what? I tried your code in a `.py` file and ran it from cmd and it blocked for me.

**Comment by c0ver** [comment]
I have the same problem in Ubuntu 20.04.1, Python 3.8.5, and pandasgui 0.2.5.1. I am running `python myscript.py` in a terminal where `myscript.py` is the basic usage example.

**Comment by adamerose** [comment]
@c0ver 
> pandasgui 0.2.5.1

I added the terminal detection & automatic blocking in 0.2.6

**Comment by technetium-43** [comment]
> @c0ver
> 
> > pandasgui 0.2.5.1
> 
> I added the terminal detection & automatic blocking in 0.2.6

Running an upgrade shows the latest available version is 0.2.5.1 FYI.

**Comment by adamerose** [comment]
> > @c0ver
> > > pandasgui 0.2.5.1
> > 
> > 
> > I added the terminal detection & automatic blocking in 0.2.6
> 
> Running an upgrade shows the latest available version is 0.2.5.1 FYI.

Oh! Thanks for pointing that out, latest versions are on PyPi now

**Comment by technetium-43** [comment]
> > > @c0ver
> > > > pandasgui 0.2.5.1
> > > 
> > > 
> > > I added the terminal detection & automatic blocking in 0.2.6
> > 
> > 
> > Running an upgrade shows the latest available version is 0.2.5.1 FYI.
> 
> Oh! Thanks for pointing that out, latest versions are on PyPi now

Awesome v0.2.7 is working now in a .py script! Thank you!

**Comment by adamerose** [comment]
Closing this since it sounds like this was just due to the PyPi release being older than the README... @forhonourlx please reopen if your issue still persists on latest release
